### ISSUE TITLE
MSE warning

### ISSUE THREAD

**Issue by githubWangHC** [sunwuzhou03/rl_learning#1]
sacwalker.py
F.mse_loss(self.critic_1(states, actions), td_target.detach()))
sacwalker.py:139: UserWarning: Using a target size (torch.Size([64, 4])) that is different to the input size (torch.Size([64, 1])). This will likely lead to incorrect results due to broadcasting. Please ensure they have the same size.

L27,log_prob = dist.log_prob(normal_sample)
Shouldn't log_prob be reduce dim?

**Comment by sunwuzhou03** [comment]
Thanks, I have fix it.
